### PR TITLE
doc_fix

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_sap/issues/7

**This PR will result in the following new package version:** v0.1.1

Non breaking

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Bugfix
- Corrects the documentation regarding which schema the models are written to.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [na] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x/ All necessary documentation and version upgrades have been applied
- [na]  docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [ ] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Just fixed the documentation in the READMe.

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ vars:
 ```  
 
 #### Change the build schema
-By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `_sap_source`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
+By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `sap`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
```
Comment (fivetran-joemarkiewicz) [inline README.md:124]: This is missing the callout for where the staging models are materialized. Can you add that relevant part in.

File: dbt_project.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 config-version: 2
 name: 'sap'
-version: '0.1.0'
+version: '0.1.1'
```
Comment (fivetran-joemarkiewicz) [inline dbt_project.yml:3]: No release will be required so we should revert this back to v0.1.0

File: integration_tests/dbt_project.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: 'sap_integration_tests'
-version: '0.1.0'
+version: '0.1.1'
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:2]: No release will be required so we should revert this back to v0.1.0

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
A few comments to be addressed before approval

File: README.md (hunk 1 above)
Comment (fivetran-reneeli) [reply README.md]: thanks Joe, updated

File: README.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ vars:
 ```  
 
 #### Change the build schema
-By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `_sap_source`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
+By default, this package builds the SAP staging models within a schema titled (`<target_schema>` + `stg_sap`) in your destination. If this is not where you would like your sap staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
```
Comment (fivetran-joemarkiewicz) [inline README.md:124]: We callout the materialization schema for the staging models, but not the transform models. Can you update this to highlight both staging and transform and also update the below yml accordingly as well.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
One final comment before approval

File: README.md (hunk 4 above)
Comment (fivetran-reneeli) [reply README.md]: @fivetran-joemarkiewicz  oh right! Updated this section.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**